apps/spruce/src/pages/distroSettings/tabs/SingleTaskDistroTab/getFormSchema.ts

```diff
@@ -17,12 +17,20 @@ export const getFormSchema = (): ReturnType<GetFormSchema> => ({
             },
             allowedTasks: {
               type: "array" as const,
-              title: "",
+              title: "Tasks",
               items: {
                 title: "Task Regex",
                 type: "string" as const,
               },
             },
+            allowedBVs: {
```

Comment:
Could we show a message that no build variants are specified for this case? (not sure if that means it defaults to all, but if it does, we should say that too)

You can use `ui:placeholder` attribute

<img width="785" alt="Screenshot 2025-03-27 at 3 08 39 PM" src="https://github.com/user-attachments/assets/ca25d04b-288d-44ec-aa71-14f743a2f596" />